product_docs/docs/postgres_for_kubernetes/1/iron-bank.mdx

```diff
@@ -90,3 +95,33 @@ directly to your Kubernetes nodes.
 
 Once you have this in place, you can apply your manifest normally with
 `kubectl apply -f`, as described in the [installation instructions](/postgres_for_kubernetes/latest/installation_upgrade/).
+
+## Deploying clusters with EPAS operands using IronBank images
+
+To deploy a cluster using the EPAS [operand](/postgres_for_kubernetes/latest/private_edb_registries/#operand-images) you must reference the Ironbank operand image appropriately in the `Cluster` resource YAML.
+For example, to deploy a PG4K Cluster using the EPAS 16 operand:
+
+1. Create or edit a `Cluster` resource YAML file with the following content:
+
+  ```yaml
+  apiVersion: postgresql.k8s.enterprisedb.io/v1
+  kind: Cluster
+  metadata:
+    name: cluster-example-full
+  spec:
+    imageName: registry1.dso.mil/ironbank/enterprisedb/edb-postgres-advanced-16@16
```

Comment:
When we provide an example to pull the images above, we use the format: 
`registry1.dso.mil/ironbank/enterprisedb/edb-postgres-advanced-16:16`

But here, we say:
```suggestion
    imageName: registry1.dso.mil/ironbank/enterprisedb/edb-postgres-advanced-16@16
```

Not sure if it is a typo that one is with 16:16 and the other with 16@16.
If it is not a typo, maybe we should call out the difference so the reader know it is correct this way?

Reply:
Fixed it.